### DISCUSSION TITLE
connection reset by peer on Jenkins build

### DISCUSSION THREAD

**Discussion by tsingh2k15** [aws/copilot-cli#3679] [Q&A] [answered]
I am using copilot with in my Jenkins CI/CD builds. While deploying a copilot service, the build often fails on this issue:

```
✘ deploy service some_service to environment some_env: deploy service: fetch service description: describe service some_service -  RequestError: send request failed

 caused by: Post "https://ecs.us-east-2.amazonaws.com/": read tcp some_ip:41934->some_ip:443: read: connection reset by peer
```

Has anyone faced the similar issue and is there a resolution? As of now my only workaround it to re-trigger the build which is not ideal.

**Comment by iamhopaul123** [comment, marked as answer]
Hello @tsingh2k15 it is probably because we make too many requests to ECS when the deployment progress tracker pulls info. Can you try to set `CI=true` as an env var to increase the pulling interval?

**Reply by tsingh2k15** [reply]
Thank you for the suggestion,  @iamhopaul123. 
 I have made the changes and will monitor the builds for few days. I will update this thread on my observations.

**Reply by tsingh2k15** [reply]
@iamhopaul123 This resolves the problem. Thank you!

**Reply by hamzaaqqa** [reply]
Hello guys 

i have same issues while deploying resources in AWS
does anyone have solution for that

**Reply by iamhopaul123** [reply]
Hello @hamzaaqqa. Does setting an env var `CI=true` not work for you?

**Reply by hamzaaqqa** [reply]
where i should use that please, i'm still stuck
i'm using terraform with jenkins to deploy resources

**Reply by iamhopaul123** [reply]
Can you set `CI=true` as an environment variable in your Jenkins CI/CD builds or wherever you run Copilot commands?

**Reply by hamzaaqqa** [reply]
actually i'm not using copilot, i'm just got the same error as you mention first 
![image](https://github.com/aws/copilot-cli/assets/23260843/ede705b3-7711-424f-bed6-caa02fdb689e)
